### DISCUSSION TITLE
倍速重置lua脚本是加在哪个文件

### DISCUSSION THREAD

**Discussion by bubbleboy11** [hooke007/mpv_PlayKit#165] [Q&A 问答] [answered]
https://github.com/hooke007/MPV_lazy/issues/136#issuecomment-1229349072，感谢大佬的回复，请问要加在哪里呢

**Comment by hooke007** [comment, marked as answer]
新建文件

**Reply by bubbleboy11** [reply]
已经在scripts/misc.lua加上了你写的代码，不知道script-opts/misc.conf要写什么内容

**Reply by hooke007** [reply]
没有要写的内容，看前三行的注释啊

**Reply by bubbleboy11** [reply]
整好了，谢谢
